compiler-rt/lib/ubsan/ubsan_value.h

```diff
@@ -103,6 +103,14 @@ class TypeDescriptor {
     /// representation is that of bitcasting the floating-point value to an
     /// integer type.
     TK_Float = 0x0001,
+    /// An _BitInt(N) type. Lowest bit is 1 for a signed value, 0 for an
+    /// unsigned
+    /// value. Remaining bits are log_2(bit_width). The value representation is
```

Comment:
nit: can you reformat the comment block to avoid the odd line break?

Reply:
Accepted.